src/framework/model.js

```diff
@@ -94,6 +100,8 @@ var RaModel = GObject.registerClass({
             return;
         this._logoGraphic = value;
         this.notify('logo-graphic');
+        this._changed = true;
+        this.notify('changed');
```

Comment:
Would be nice to have a `_setChanged()` function to avoid repeating these two lines over and over

Reply:
I went with a different approach after all, without repeating this.